src/calculate/pass_3.rs

```diff
@@ -17,18 +17,19 @@ const OPERATOR_HAS_PENALTY: [EffectOperator; 5] = [
 
 pub struct PassThree {}
 
-struct Cache {
-    hull: BTreeMap<i32, f64>,
-    char: BTreeMap<i32, f64>,
-    structure: BTreeMap<i32, f64>,
-    target: BTreeMap<i32, f64>,
-    items: BTreeMap<usize, BTreeMap<i32, f64>>,
-    charge: BTreeMap<usize, BTreeMap<i32, f64>>,
-    skills: BTreeMap<usize, BTreeMap<i32, f64>>,
+#[derive(Default)]
```

Comment:
I already cherry-picked this "default" part into the main. It is just a good change, so I didn't want to delay that.